src/Components/MinimalComponentModel.cs

```diff
@@ -193,6 +195,31 @@ public ProjectFolderItemType Type
         /// </summary>
         public bool CanTranslate { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether this component is followed.
+        /// </summary>
+        public bool IsFollowed { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this component is followed by current user.
+        /// </summary>
+        public bool IsFollowedByCurrentUser { get; set; }
```

Comment:
Can this getter be derived to return value based on the current context user? Otherwise, somewhere else this needs to be set I guess.

Reply:
It could but I find it much easier here. There can be many followed by users for a single component. So we would need to do a filter on them looking for a match to current user.  If you don't see an efficiency flaw here I'd like to keep as is.

Reply:
Sure, if that makes sense to you. I just wanted to trigger that thought just in case if you didn't realize that use case.